src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -74,7 +74,7 @@ export SENTRY_PROPERTIES=sentry.properties
 
 [[ $SENTRY_INCLUDE_NATIVE_SOURCES == "true" ]] && INCLUDE_SOURCES_FLAG="--include-sources" || INCLUDE_SOURCES_FLAG=""
 SENTRY_CLI="../node_modules/@sentry/cli/bin/sentry-cli"
-$SENTRY_CLI debug-files upload "$INCLUDE_SOURCES_FLAG" "${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.app"
+$SENTRY_CLI debug-files upload "$INCLUDE_SOURCES_FLAG" "$DWARF_DSYM_FOLDER_PATH"
```

Comment:
@krystofwoldrich did you check if there's more usages of `debug-files upload` or `upload-dif` or `upload-dsym` in the RN docs?